release-notes/Breaking_changes/Breaking_changes.md

```diff
@@ -8,14 +8,27 @@ The following breaking changes have been introduced in recent DataMiner releases
 
 | Release note ID | Release version(s) | Description |
 |--|--|--|
+| [34895](xref:General_Feature_Release_10.3.1#dashboards-app--low-code-apps---gqi-certain-cell-values-in-a-gqi-query-result-would-incorrect-include-the-object-type-id_34895) | DataMiner 10.2.0 [CU1]/10.3.1 | Previously, cell values of GQI result rows for DomInstanceIds, DomDefinitionIds, ProfileInstanceIds, and ProfileDefinitionIds contained both the display value and the object type. Now the GUID value is used as the cell value. |
```

Comment:
'contained both the display value of the GUID and the object type. Now only the display value of the GUID is used.' is a bit more clear I think.